README.md

```diff
@@ -58,26 +17,97 @@ To run the application without specifying the configuration file path, save the
-- macOS: $HOME/Library/Application Support/com.cluml.github-dashboard-server/config.toml
+- macOS: $HOME/Library/Application
+  Support/com.cluml.github-dashboard-server/config.toml
 
 The web server will use the address value specified in the configuration file.
 
 ### Accessing the Web Interface
 
 - Open <https://localhost:8000> in your browser to run the app.
-- Visit <https://localhost:8000/graphql/playground> to access the GraphQL playground.
+- Visit <https://localhost:8000/graphql/playground> to access the GraphQL
+  playground.
 
-## FLAGS
+### Flags
 
 - `-h`, `--help`: Displays help information.
 - `-V`, `--version`: Displays version information.
 
-## OPTION
+### Options
+
+- `config_file`: The path to the TOML file containing server configuration
+  details.
+
+## Configuration
+
+In the configuration file, you can specify the following options:
+
+### `[web]`
+
+| Field     | Description                                 | Required | Default |
+| --------- | ------------------------------------------- | -------- | ------- |
+| `address` | The address of web server                   | Yes      | -       |
+| `key`     | The TLS key path for the web server         | Yes      | -       |
+| `cert`    | The TLS certificate path for the web server | Yes      | -       |
+
+### `[[repositories]]`
 
-- `config_file`: The path to the TOML file containing server configuration details.
+| Field   | Description                        | Required | Default |
+| ------- | ---------------------------------- | -------- | ------- |
+| `owner` | The owner of the GitHub repository | Yes      | -       |
+| `name`  | The name of the GitHub repository  | Yes      | -       |
 
-## GitHub GraphQL API Testing
+### `[certification]`
 
-The GitHub GraphQL API used in this project was last tested on 2025-05-19. It is
-advisable to regularly review [breaking changes](https://docs.github.com/en/graphql/overview/breaking-changes)
-in the GitHub GraphQL API.
+<!-- markdownlint-disable -->
+
+| Field   | Description                                          | Required | Default |
+| ------- | ---------------------------------------------------- | -------- | ------- |
+| `token` | A GitHub fine-grained personal access token          | Yes      | -       |
+| `ssh`   | The path to SSH private key for GitHub code checkout | Yes      | -       |
+
+<!-- markdownlint-enable -->
+
+- `token`: The
+  [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-token)
+  's lifetime should be less than one year for security purposes. Minimum
+  required permissions are as follows:
+  - Repository: Access to all repositories
+  - Issues: Read-only access
+  - Pull Requests: Read-only access
+- `ssh`: To provide an SSH passphrase, set the `SSH_PASSPHRASE` environment
+  variable.
+
+### `[database]`
+
+<!-- markdownlint-disable -->
```

Comment:
Since this disables all lints at once, it would be more minimal and optimal to specify only the lint(s) that actually need to be silenced.